modules/olm-installing-from-operatorhub-using-web-console.adoc

```diff
@@ -17,27 +17,24 @@ permissions.
 
 . Navigate in the web console to the *Catalog → Operator Hub* page.
 
-. Choose the Operator you want to install from the list of available Operators (*etcd* for this example).
-A pop-up window appears, warning you that you are installing a Community Operator.
+. Scroll or type a keyword into the `Filter by keyword` box (in this case, Couchbase) to find the Operator you want.
```

Comment:
All GUI items need to be enclosed with ``. Parentheses are still bad.

Reply:
Okay on the GUI items. Parentheses are a connected to my soul. I'm sorry if they hurt you. ;-)